index.html

```diff
@@ -59,7 +59,7 @@ <h2 class="results">results for 'orange'</h2>
         </div>
         <ul class="search-results"></ul>
       </div>
-      <div data-view="details" class="individual-anime hidden"></div>
+      <div data-view="details" id="what" class="individual-anime hidden"></div>
```

Comment:
`what` isn't a very descriptive `id` name. Something like `id="individual-anime"` would be better